rl_zoo3/vec_env_wrappers.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Iterable, List, Optional, Sequence, Type, Union
+
+import gym
+import numpy as np
+from envpool.python.protocol import EnvPool
+from gym import spaces
+from stable_baselines3.common.vec_env import VecEnv
+from stable_baselines3.common.vec_env.base_vec_env import VecEnvObs, VecEnvStepReturn
+
+# Used when we want to access one or more VecEnv
+VecEnvIndices = Union[None, int, Iterable[int]]
```

Comment:
we don't have that type hint defined in SB3?

Reply:
Yes, in `base_with_env`. I don't remember why I redefined it here. I probably should have imported it